.github/workflows/python-package.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["3.8", "3.9", "3.10", "3.11"]
+        python-version: ["3.9", "3.10", "3.11"]
```

Comment:
Please lets  keep support for python3.8

Reply:
@pateash 3.8 will be EOL in one month (oct 2024). That is why I have removed it. Do we have customers actively using 3.8 that we know for sure?

https://devguide.python.org/versions/

Reply:
There might be a few customers using 3.8, we have to keep it going for some time.

Reply:
even if we have customers using 3.8 they should be updating. Already some tests were failing on python3.8. 

we should not support out-of-date python unless we are certain that there is a customer who cannot upgrade their python.

Reply:
It makes sense but I am not confident that all of our customer (who are on legacy platforms like cloudera) will be able to do that.

if we want to remove support for 3.8 we should have a major release for PBT.
@jainabhinav and @prongs  for any more comments.